.gitignore

```diff
@@ -17,6 +17,9 @@
 # emacs
 flycheck*
 
+## Visual code settings
+.vscode/
+
```

Comment:
Actually, when is the last time you pulled from main? I believe I already introduced this to .gitignore 4 days ago:

https://github.com/pyansys/pymapdl/blob/180a6647527da0b7ddee68ccadcb7d89d84861e5/.gitignore#L86-L87

Reply:
ups..... you are right!

Reply:
Useless PR ever I guess.........